drivers/SmartThings/philips-hue/src/handlers/attribute_emitters.lua

```diff
@@ -125,30 +112,120 @@ local function _emit_light_events_inner(light_device, light_repr)
   end
 end
 
+function AttributeEmitters.connectivity_update(child_device, zigbee_status)
+  if zigbee_status.status == "connected" then
+    child_device.log.info_with({hub_logs=true}, "Light status event, marking device online")
```

Comment:
Should "Light" in the log message be changed to something more general or is this only called for lights?

Reply:
Yes, but I wanted to minimize churn. I was going to do a docstring/logging pass as a standalone custodial PR: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/feat/device/motion-and-contact-sensors/drivers/SmartThings/philips-hue/PROGRESS.md#miscellaneouscustodial